planet/cli/data.py

```diff
@@ -64,11 +64,12 @@ def assets_to_filter(ctx, param, assets: List[str]) -> Optional[dict]:
 
 def check_item_types(ctx, param, item_types) -> Optional[List[dict]]:
     # Set difference between given item types and all item types
-    set_diff = set([item.lower() for item in item_types]) - set(
+    invalid_item_types = set([item.lower() for item in item_types]) - set(
         [a.lower() for a in ALL_ITEM_TYPES])
-    if set_diff:
-        raise click.BadParameter(
```

Comment:
I think we want to still raise `click.BadParameter` here. Ultimately, the issue *is* still a bad parameter. And click handles `click.BadParameter` differently from other errors.

Reply:
Good call!

Reply:
If we use `specs.validate_item_type()` to validate the item types, as you suggested in [your comment](https://github.com/planetlabs/planet-client-python/pull/695/files/4935b11afab9a5c731831f6930d5bcc2ab5783fd#r977970613), then we'll need to raise the `SpecificationException`, as is raised in the `specs` module.